nyc_data/ppe/migrations/0006_auto_20200410_1635.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.5 on 2020-04-10 16:35
```

Comment:
Ahh, so I think I'm starting to understand the purpose of the `.py` files within the migration folder. 

Whenever `python manage.py migrate` gets ran, django checks for any changes in any of the models. If it sees changes, it logs them in these files. These files are then committed into the project to keep a record of all changes within the database over time.

Am I on the right track?

Reply:
Mostly. These files define how to go from an empty database (or an earlier version) to a database with all the tables defined.

Django can autogenerate these when you modify your models.

It creates a table in your database to track which migrations have already been applied